packages/kbn-es/src/cluster.js

```diff
@@ -325,6 +325,11 @@ exports.Cluster = class Cluster {
     this._process.stdout.on('data', (data) => {
       const lines = parseEsLog(data.toString());
       lines.forEach((line) => {
+        if (line.message.includes('license mode is')) {
```

Comment:
This line happens twice, resulting in two ready logs.

How about on the first log line with a `publish_address`?